mobile/app.json

```diff
@@ -3,7 +3,7 @@
     "name": "dsocial",
     "slug": "dsocial",
     "platforms": ["ios", "android"],
-    "version": "1.0.0",
+    "version": "1.9.3",
```

Comment:
We shouldn't change the version number here because it won't be updated automatically.

Reply:
https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md